docs/services/safety_manual_check.rst

```diff
@@ -1,19 +1,34 @@
 Safety Manual Check
 ===================
 
+A service that allows the user to manually check the safety of the testbed through the :ref:`safety_monitor` service.
+
+This service has a property called ``value`` whose initial value is read from the configuration file, and it can be set to any value by the user.
+It is continuously submitted to the only datastream ``check``.
+
+By setting the value to something new, the safety monitor picks up on the new value on the datastream ``check`` and it should react accordingly.
 
 Configuration
 -------------
 
+.. code-block:: YAML
 
+    safety_manual_check:
+      service_type: safety_manual_check
+      requires_safety: true
+
+      dtype: int
+      initial_value: 5
 
 Properties
 ----------
+``value``:
```

Comment:
Oh I also need to fill these in, whoops.

Reply:
Ah I didn't do that because I didn't fully understood how this service works... Where does the `self.check_value[0]` in it come from?

Reply:
This was a bug, fixed with @ehpor.